ckanext/ontario_theme/templates/internal/package/read.html

```diff
@@ -84,7 +84,9 @@ <h1 itemprop="name">
           {% snippet "package/snippets/tags.html", tags=pkg.tags %}
         {% endblock package_tags %}
 
-        {% snippet "package/snippets/report_an_error.html" %}
+        {% if pkg.resources %}
```

Comment:
If a dataset is not open but has a resource, the banner still shows up. Changing this line to something like `if pkg.resources and the package's "access_level" == open` will prevent the banner from showing up